packages/common/container-definitions/src/loader.ts

```diff
@@ -275,13 +391,19 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
 
     /**
      * The server provided ID of the client.
-     * Set once this.connectionState === ConnectionState.Connected is true, otherwise undefined
+     *
+     * Set once {@link IContainer.connectionState} is {@link (ConnectionState:namespace).Connected},
+     * otherwise undefined.
+     *
      * @alpha
```

Comment:
not sure what @alpha means here :)

Reply:
It's a good question. I erred on the side of leaving this unchanged, but to my knowledge we have no guidance / policy on use of `@alpha`.  This is also clearly used. I'll go ahead and remove it.